### ISSUE TITLE
Still can't pip install minari

### ISSUE THREAD

**Issue by sryu1** [Farama-Foundation/Minari#72]
```
PS C:\Users\noahs> py -m pip install --upgrade minari
Collecting minari
  Using cached minari-0.3.1-py3-none-any.whl (30 kB)
Requirement already satisfied: numpy>=1.21.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from minari) (1.23.5)
Collecting h5py==3.7.0 (from minari)
  Using cached h5py-3.7.0.tar.gz (392 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting tqdm>=4.65.0 (from minari)
  Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
Requirement already satisfied: typing-extensions==4.4.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from minari) (4.4.0)
Collecting google-cloud-storage==2.5.0 (from minari)
  Using cached google_cloud_storage-2.5.0-py2.py3-none-any.whl (106 kB)
Collecting typer[all]==0.7.0 (from minari)
  Using cached typer-0.7.0-py3-none-any.whl (38 kB)
Requirement already satisfied: gymnasium>=0.28.1 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from minari) (0.28.1)
Requirement already satisfied: google-auth<3.0dev,>=1.25.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from google-cloud-storage==2.5.0->minari) (2.16.2)
Collecting google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 (from google-cloud-storage==2.5.0->minari)
  Using cached google_api_core-2.11.0-py3-none-any.whl (120 kB)
Collecting google-cloud-core<3.0dev,>=2.3.0 (from google-cloud-storage==2.5.0->minari)
  Using cached google_cloud_core-2.3.2-py2.py3-none-any.whl (29 kB)
Collecting google-resumable-media>=2.3.2 (from google-cloud-storage==2.5.0->minari)
  Using cached google_resumable_media-2.5.0-py2.py3-none-any.whl (77 kB)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from google-cloud-storage==2.5.0->minari) (2.28.2)
Requirement already satisfied: click<9.0.0,>=7.1.1 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from typer[all]==0.7.0->minari) (8.1.3)
Requirement already satisfied: colorama<0.5.0,>=0.4.3 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from typer[all]==0.7.0->minari) (0.4.6)
Collecting shellingham<2.0.0,>=1.3.0 (from typer[all]==0.7.0->minari)
  Using cached shellingham-1.5.0.post1-py2.py3-none-any.whl (9.4 kB)
Requirement already satisfied: rich<13.0.0,>=10.11.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from typer[all]==0.7.0->minari) (12.6.0)
Requirement already satisfied: jax-jumpy>=1.0.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from gymnasium>=0.28.1->minari) (1.0.0)
Requirement already satisfied: cloudpickle>=1.2.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from gymnasium>=0.28.1->minari) (2.2.1)
Requirement already satisfied: farama-notifications>=0.0.1 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from gymnasium>=0.28.1->minari) (0.0.4)
Collecting googleapis-common-protos<2.0dev,>=1.56.2 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-storage==2.5.0->minari)
  Using cached googleapis_common_protos-1.59.0-py2.py3-none-any.whl (223 kB)
Requirement already satisfied: protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-storage==2.5.0->minari) (4.22.1)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from google-auth<3.0dev,>=1.25.0->google-cloud-storage==2.5.0->minari) (5.3.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from google-auth<3.0dev,>=1.25.0->google-cloud-storage==2.5.0->minari) (0.2.8)
Requirement already satisfied: six>=1.9.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from google-auth<3.0dev,>=1.25.0->google-cloud-storage==2.5.0->minari) (1.16.0)
Requirement already satisfied: rsa<5,>=3.1.4 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from google-auth<3.0dev,>=1.25.0->google-cloud-storage==2.5.0->minari) (4.9)
Collecting google-crc32c<2.0dev,>=1.0 (from google-resumable-media>=2.3.2->google-cloud-storage==2.5.0->minari)
  Using cached google_crc32c-1.5.0-cp311-cp311-win_amd64.whl (27 kB)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from requests<3.0.0dev,>=2.18.0->google-cloud-storage==2.5.0->minari) (3.0.1)
Requirement already satisfied: idna<4,>=2.5 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from requests<3.0.0dev,>=2.18.0->google-cloud-storage==2.5.0->minari) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from requests<3.0.0dev,>=2.18.0->google-cloud-storage==2.5.0->minari) (1.26.14)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from requests<3.0.0dev,>=2.18.0->google-cloud-storage==2.5.0->minari) (2022.12.7)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from rich<13.0.0,>=10.11.0->typer[all]==0.7.0->minari) (0.9.1)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from rich<13.0.0,>=10.11.0->typer[all]==0.7.0->minari) (2.14.0)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in c:\users\noahs\appdata\local\programs\python\python311\lib\site-packages (from pyasn1-modules>=0.2.1->google-auth<3.0dev,>=1.25.0->google-cloud-storage==2.5.0->minari) (0.4.8)
Building wheels for collected packages: h5py
  Building wheel for h5py (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for h5py (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [72 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-311
      creating build\lib.win-amd64-cpython-311\h5py
      copying h5py\h5py_warnings.py -> build\lib.win-amd64-cpython-311\h5py
      copying h5py\ipy_completer.py -> build\lib.win-amd64-cpython-311\h5py
      copying h5py\version.py -> build\lib.win-amd64-cpython-311\h5py
      copying h5py\__init__.py -> build\lib.win-amd64-cpython-311\h5py
      creating build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\attrs.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\base.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\compat.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\dataset.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\datatype.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\dims.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\files.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\filters.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\group.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\selections.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\selections2.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\vds.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      copying h5py\_hl\__init__.py -> build\lib.win-amd64-cpython-311\h5py\_hl
      creating build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\common.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\conftest.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_attribute_create.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_attrs.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_attrs_data.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_base.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_big_endian_file.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_completions.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_dataset.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_dataset_getitem.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_dataset_swmr.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_datatype.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_dimension_scales.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_dims_dimensionproxy.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_dtype.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_errors.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_file.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_file2.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_file_alignment.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_file_image.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_filters.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_group.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_h5.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_h5d_direct_chunk.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_h5f.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_h5o.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_h5p.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_h5pl.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_h5t.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_objects.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_selections.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\test_slicing.py -> build\lib.win-amd64-cpython-311\h5py\tests
      copying h5py\tests\__init__.py -> build\lib.win-amd64-cpython-311\h5py\tests
      creating build\lib.win-amd64-cpython-311\h5py\tests\data_files
      copying h5py\tests\data_files\__init__.py -> build\lib.win-amd64-cpython-311\h5py\tests\data_files
      creating build\lib.win-amd64-cpython-311\h5py\tests\test_vds
      copying h5py\tests\test_vds\test_highlevel_vds.py -> build\lib.win-amd64-cpython-311\h5py\tests\test_vds
      copying h5py\tests\test_vds\test_lowlevel_vds.py -> build\lib.win-amd64-cpython-311\h5py\tests\test_vds
      copying h5py\tests\test_vds\test_virtual_source.py -> build\lib.win-amd64-cpython-311\h5py\tests\test_vds
      copying h5py\tests\test_vds\__init__.py -> build\lib.win-amd64-cpython-311\h5py\tests\test_vds
      copying h5py\tests\data_files\vlen_string_dset.h5 -> build\lib.win-amd64-cpython-311\h5py\tests\data_files
      copying h5py\tests\data_files\vlen_string_dset_utc.h5 -> build\lib.win-amd64-cpython-311\h5py\tests\data_files
      copying h5py\tests\data_files\vlen_string_s390x.h5 -> build\lib.win-amd64-cpython-311\h5py\tests\data_files
      running build_ext
      Loading library to get build settings and version: hdf5.dll
      error: Unable to load dependency HDF5, make sure HDF5 is installed properly
      error: Could not find module 'hdf5.dll' (or one of its dependencies). Try using the full path with constructor syntax.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for h5py
Failed to build h5py
ERROR: Could not build wheels for h5py, which is required to install pyproject.toml-based projects
```

**Comment by rodrigodelazcano** [comment]
Hey @rodrigodelazcano if you have time will you be able to install Minari from source and check if it works now? 
I merged the following PR to solve the issue #73

**Comment by sryu1** [comment]
yep, installing with source works.
